src/appleseed/renderer/modeling/scene/assembly.cpp

```diff
@@ -56,6 +56,8 @@ namespace renderer
 // Assembly class implementation.
 //
 
+const char* Model = "assembly";
```

Comment:
And indeed it would make sense to rename `Assembly` to `GenericAssembly` if we start to have many kinds of assemblies.